Source/Blazorise/Components/Droppable/DropZone.razor.cs

```diff
@@ -107,53 +140,86 @@ private void OnContainerTransactionStarted( object sender, DraggableTransaction<
             InvokeAsync( StateHasChanged );
         }
 
-        private void OnContainerTransactionEnded( object sender, EventArgs e )
+        private void OnContainerTransactionEnded( object sender, DraggableTransactionEnded<TItem> e )
         {
-            itemOnDropZone = false;
+            dragCounter = 0;
 
             if ( GetApplyDropClassesOnDragStarted() )
             {
                 dropAllowed = false;
             }
 
+            if ( e.Success )
+            {
+                if ( e.OriginDropZoneName == Name && e.DestinationDropZoneName != e.OriginDropZoneName )
```

Comment:
Could probably evaluate only once like:

var isOriginDropZone = blabla
var isDestinationDropZone = blabla

And then just use in each if clause, still method is so short and logic so simple... only a matter of preference

Reply:
I will leave it as it is now as there is reusage in the current implementation.